padglobal/padglobal.py

```diff
@@ -364,6 +365,28 @@ async def eventswap(self, ctx):
         json.dump(self.c_commands, open(self.file_path, 'w+'))
         await ctx.send(f"Okay, I've updated {inline('lastevent')} and {inline('currentevent')} accordingly,"
                        f" and cleared {inline('nextevent')}.")
+    
+    @padglobal.command()
+    async def rename(self, ctx, old_name: str , new_name: str):
```

Comment:
You don't need to type strings in commands!  It's actually slower if you do :0  You can leave them untyped (remove the : str)